pkg/ceph/osd/health_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2017 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package osd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/rook/rook/pkg/ceph/mon"
+	"github.com/rook/rook/pkg/clusterd"
+	exectest "github.com/rook/rook/pkg/util/exec/test"
+	"github.com/rook/rook/pkg/util/kvstore"
+	"github.com/rook/rook/pkg/util/proc"
+	"github.com/stretchr/testify/assert"
+)
+
+func createTAgent(t *testing.T, devices, configDir string, storeConfig *StoreConfig) (*OsdAgent, *exectest.MockExecutor) {
```

Comment:
this looks identical to the already existing `createTestAgent` func in this same package.  is there a need for this extra function too?